### PR TITLE
Update docs for master to remove Python 2 references

### PR DESCRIPTION
**Pull request by jlin27**
Fix compile error from original PR in jit_language_references.rst: https://github.com/pytorch/pytorch/pull/36114

Full details in task: https://our.internmc.facebook.com/intern/tasks/?t=64776265

With pytroch 1.5+ we remove python2 support from PyTorch. All documentation under docs/ and on the pytorch.org website needs to remove Python 2 references.

### CODE REVIEW

File: docs/source/notes/multiprocessing.rst [hunk 1]
```diff
@@ -27,9 +27,7 @@ others that require asynchronous operation.
 CUDA in multiprocessing
 -----------------------
 
-The CUDA runtime does not support the ``fork`` start method. However,
-:mod:`python:multiprocessing` in Python 2 can only create subprocesses using
-``fork``. So Python 3 and either ``spawn`` or ``forkserver`` start method are
+The CUDA runtime does not support the ``fork`` start method. In Python 3, either the ``spawn`` or ``forkserver`` start method are
```
Comment (xuhdev) [inline docs/source/notes/multiprocessing.rst:30]: Does it sound a bit weird to say "in Python 3" here? Do you think we can simply remove the phrase?

Comment (jlin27) [reply docs/source/notes/multiprocessing.rst]: Removed reference to Python 3 here. Thanks

**Review by orionr — APPROVED** [review]
LGTM!

**Comment by orionr** [comment]
Please fix clang-format if applicable?

**Comment by jlin27** [comment]
Reached out to Meghan Lele who recently made some changes related to clang-format in the affected file (torch/csrc/jit/tensorexpr/cuda_codegen.cpp) and she thinks this is the PR that broke it: https://github.com/pytorch/pytorch/pull/36306

And since our changes are in an unrelated directory is should be okay to land.

She alerted the people who usually work in `jit/tensorexpr` and hopefully they fix it soon.

Will go ahead and merge

**Pull request closed without merging**